tests/data/mtx/bad_format_has_space.mtx

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Is this file bad because of this first empty line or the line after the comment lines? From the code, it sounds like we're checking for the first line. Is line 5 important in these tests, if not, can we remove it to be explicit about what we are testing?

Reply:
It keeps checking until a valid or invalid character is reached. If neither happens it's an empty file.